_rules/auto-update-text-efbfc7.md

```diff
@@ -22,16 +22,21 @@ acknowledgements:
 
 The rule applies to any [visible][] [HTML element][] in an [HTML document][] if:
```

Comment:
Suggesting reformulating the enumeration (splitting conditions on the element and on the changes, and naming conditions as Wilco is introducing in https://github.com/act-rules/act-rules.github.io/pull/1171/files#diff-ba71ed44d1a01755345d5f6362432855R34-R39) in the line of:

The rule applies to any element in a document if the element:
- **(not alone)** is not the only content; and
- **(change)** has a `innerText` that changes; and
- **(no child change)** does not have children …; and
- is part of a document:
  - **(ready)** whose readiness is complete when the change happens; and
  - **(inactive)** is associated to a window with infitine timestamp.

I think at least the nested list is good to avoid the complex reverse relatives to qualify the document or the window.